### ISSUE TITLE
Search.php - hardcoded PL in js, leak of regions outside of PL

### ISSUE THREAD

**Issue by GoogleCodeExporter** [opencaching/opencaching-pl#26]
```
When pressing "Look for a cache" button on the left menu and leave all options 
like it is (this is only upper part of the screen) and press the button 
"Search" (above Search by cache name). Now i see 44 caches on oc.nl. This 
should be correct. But i like to see all caches (including mine). Than i select 
on top "show my caches" and press search again. Now i see only one cache 
located in Poland?

When going back to search page the Country has changed to Poland. Change it to 
all countries and press search again. The result again only one cache located 
in Poland.

Back on search page the country is again changed in Poland. When i change it to 
all countries and press Region i see Polish regions.

In short, searching for caches in all countries shows only Poland caches.

```

Original issue reported on code.google.com by `henricus...@gmail.com` on 26 Jan 2014 at 9:41
